src/webui/www/private/views/preferences.html

```diff
@@ -226,6 +226,28 @@
             QBT_TR(Tip: Encapsulate parameter with quotation marks to avoid text being cut off at whitespace (e.g., "%N"))QBT_TR[CONTEXT=OptionsDialog]
         </div>
     </fieldset>
+
+    <fieldset class="settings">
+        <legend>
+            RSS
+        </legend>
+        <div class="formRow">
+            <label for="rssProcessingEnabledCheckbox">QBT_TR(Enable fetching RSS feeds)QBT_TR[CONTEXT=OptionsDialog]</label>
+            <input type="checkbox" id="rssProcessingEnabledCheckbox">
+        </div>
+        <div class="formRow">
+            <label for="rssRefreshIntervalNumber">QBT_TR(Feeds refresh interval)QBT_TR[CONTEXT=OptionsDialog]</label>
```

Comment:
Please add colon at the end: `Feeds refresh interval:`

Reply:
Fixed.